tee-worker/litentry/core/assertion-build/src/nodereal/bnb_domain/mod.rs

```diff
@@ -0,0 +1,289 @@
+			.into_iter()
+			.filter(|info| digit_domain_type.is_member(&info.name))
+			.collect();
+
+		Ok(infos)
+	}
+}
+
+#[cfg(test)]
+mod tests {
+	use super::Domains;
+	use litentry_primitives::BnbDigitDomainType;
+
+	const RESPONSE: &'static str = r#"
+		{
+			"0xr4b0bf28adfcee93c5069982a895785c9231c1fe1": [
+				{
+					"nodeHash": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"bind": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"name": "1",
+					"expires": "2024-08-24T00:36:44Z"
+				},
+				{
+					"nodeHash": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"bind": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"name": "2",
+					"expires": "2032-08-24T00:15:56Z"
+				}
+			],
+			"0xr4b0bf28adfcee93c5069982a895785c9231c1fe2": [
+				{
+					"nodeHash": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"bind": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"name": "3",
+					"expires": "2023-08-24T20:36:26Z"
+				},
+				{
+					"nodeHash": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"bind": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"name": "4",
+					"expires": "2023-08-24T20:35:59Z"
+				},
+				{
+					"nodeHash": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"bind": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"name": "win",
+					"expires": "2023-08-24T20:38:29Z"
+				}
+			],
+			"0xr4b0bf28adfcee93c5069982a895785c9231c1fe3": [
+				{
+					"nodeHash": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"bind": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"name": "5",
+					"expires": "2024-08-24T06:33:32Z"
+				},
+				{
+					"nodeHash": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"bind": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"name": "6",
+					"expires": "2024-08-24T07:57:41Z"
+				},
+				{
+					"nodeHash": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"bind": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"name": "7",
+					"expires": "2023-09-16T19:36:14Z"
+				}
+			],
+			"0xr4b0bf28adfcee93c5069982a895785c9231c1fe4": [
+				{
+					"nodeHash": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"bind": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"name": "8",
+					"expires": "2028-09-18T13:35:38Z"
+				}
+			],
+			"0xr4b0bf28adfcee93c5069982a895785c9231c1fe5": [
+				{
+					"nodeHash": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"bind": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"name": "9",
+					"expires": "2023-09-03T08:35:40Z"
+				}
+			],
+			"0xr4b0bf28adfcee93c5069982a895785c9231c1fe6": [
+				{
+					"nodeHash": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"bind": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"name": "000",
+					"expires": "2024-10-30T18:40:51Z"
+				},
+				{
+					"nodeHash": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"bind": "0xr4b0bf28adfcee93c5069982a895785c9231c1fe",
+					"name": "999",
+					"expires": "2024-03-19T18:16:59Z"
```

Comment:
This hardcoded date will make `non_expired_domain_infos_works` test case fail in 4 months. Use `Instant::now()` with offset to make it work at any time in future.

Reply:
@zhouhuitian FYI